.github/CODEOWNERS

```
@@ -0,0 +1,10 @@
+# CODEOWNERS: https://help.github.com/articles/about-codeowners/
+
+# Everything goes through the following "global owners" by default.
+# Unless a later match takes precedence, these three will be
+# requested for review when someone opens a PR.
+# Note that the last matching pattern takes precedence, so
+# global owners are only requested if there isn't a more specific
+# codeowner specified below. For this reason, the global codeowners
+# are often repeated in package-level definitions.
+*       @liamsi @renaynay @Wondertan
```

Let's add @mattdf as well? So he can chime in from time to time? 
```suggestion
*       @liamsi @renaynay @Wondertan @mattdf 
```

---

@mattdf do you consent to this?

---

:+1: